lib/uname.js

```diff
@@ -0,0 +1,43 @@
+var getUname = require('node-uname')
+var decodeArguments = require('./common').decodeArguments
+
+module.exports = uname
+
+function uname(args) {
+    var params = decodeArguments(args)
+    var info = getUname()
+
+    if (params.a || params.all) {
+        return [info.sysname, info.nodename, info.release, info.version, info.machine].join(' ')
```

Comment:
I'm not sure if I quite understand.... Are you saying that I should implement the .inspect() method on the object returned by node-uname?